doc/user_guide/dynamo0p3.rst

```diff
@@ -2455,32 +2457,33 @@ Run-time Checks
 
 PSyclone performs static consistency checks where possible. When this
 is not possible PSyclone can generate run-time checks. As there may be
-performance costs associated with run-time checks this can be swiched
-on or off by the `RUN-TIME-CHECKS` option in the configuration file.
+performance costs associated with run-time checks they may be switched
+on or off by the `RUN_TIME_CHECKS` option in the configuration file.
 
-Currently run-time checks can be generated to
+Currently run-time checks can be generated to:
 
 1) check that a field with a read-only function space (see section
    :ref:`lfric-ro-function-space`) is not modified by a kernel. This is
-   enforced by checking that all fields whose data is passed into a
-   kernel, where the kernel metadata specifies that the field's data will
-   be modified, are not read-only fields. The other check that needs to
-   be performed for read-only function spaces is that the halo is valid
-   when the field's halo is accessed. This is currently enforced within
-   the LFRic infrastructure halo exchange call (that the PSyclone LFRic
-   API places at appropriate locations). If the halo is valid then the
-   halo exchange will not be called. Therefore, if the halo exchange is
-   called with a field on a read-only function space then the
-   infrastructure reports an error.
+   enforced by checking that all fields that are marked (in kernel
+   metadata) as being updated by a kernel are not on a read-only function
+   space. A second check that is required for fields on read-only
+   function spaces is to ensure that fields halo is valid when it is
```

Comment:
"...ensure that the halo is clean before it is accessed." and below s/halo is valid/halo is clean/

Reply:
Modified.